src/pages/NoteCollectionPage.tsx

```diff
@@ -133,7 +134,8 @@ const NoteCollectionPage: React.FC = () => {
     orderBy,
     filters,
   )
-  const { foundNote, loadingFoundNote } = useFindMemberNoteByNoteId(activeMemberNoteId || '')
+  const { memberNote, loadingFoundNote } = useFindMemberNoteByNoteId(activeMemberNoteId || '')
```

Comment:
could be more Simplify. suck like useMemberNoteById

Reply:
Originally, variable names included verbs. 
It's a good suggestion. I'll change it to 'memberNote' and 'loadingMemberNote'

Reply:
it can also be like this:
```
const foundMemberNote = useFoundMemberNote()
foundMemberNote.data
foundMemberNote.isLoading
```